hackathon_site/hardware/test_api.py

```diff
@@ -194,3 +194,164 @@ def test_hardware_get_success(self):
         data = response.json()
 
         self.assertEqual(expected_response, data["results"])
+
+
+class HardwareListTestCase(SetupUserMixin, APITestCase):
```

Comment:
Also, just to be consistent with the test case that's getting merged in and the rest of them, rename this to `HardwareListViewTestCase`